database/migrations/2024_06_24_074114_create_wiki_entity_imports_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('wiki_entity_imports', function (Blueprint $table) {
+            $table->id();
+            $table->timestamps();
+
+            $table->unsignedInteger('wiki_id');
+            $table->foreign('wiki_id')->references('id')->on('wikis');
+
+            $table->enum('status', ['pending', 'failed', 'success']);
+            $table->timestamp('started_at')->nullable();
+            $table->timestamp('finished_at')->nullable();
+            $table->json('payload')->nullable();
```

Comment:
Technically, storing this information is not necessary, but I thought it might be very helpful when debugging failed runs while rolling out this feature. Once things work, we might think about dropping it again.

Reply:
it might even be useful to display this information to the user, in case they forgot or are confused if they did the importing and on which wiki, etc.